ooiui/templates/science/index.html

```diff
@@ -9,32 +9,8 @@
   <!-- d3 doesn't work when used with "use strict" -->
   <script src="/lib/d3/d3.js" type="text/javascript"></script>
   
-  <!-- Map Stuff -->
-  <link rel="stylesheet" href="/lib/leaflet/dist/leaflet.css"/>
 
-  <link rel="stylesheet" href="/lib/Leaflet.awesome-markers/dist/leaflet.awesome-markers.css">
-
-  <link rel="stylesheet" href="/css/common/index.css">
-
-  <!-- Marker Cluster -->
-  <link rel="stylesheet" href="/lib/leaflet.markercluster/dist/MarkerCluster.css" />
-  <link rel="stylesheet" href="/lib/leaflet.markercluster/dist/MarkerCluster.Default.css" />
-  <!-- Marker Cluster -->
-
-  <!--
-  Date Time
-  -->    
-  <link rel="stylesheet" href="/lib/bootstrap3-datetimepicker/build/css/bootstrap-datetimepicker.min.css">
-
-
-  <link rel="stylesheet" href="/css/common/plot.css">
-  
-  <!-- Everything for fancy tree -->
-  <link href="/lib/fancytree/dist/skin-bootstrap/ui.fancytree.css" rel="stylesheet" type="text/css" class="skinswitcher">
-
-  <!-- end fancy-tree -->
-
-  <link rel="stylesheet" href="/css/common/toc_menu.css" />
+  <link href="/css/compiled/science.css" rel="stylesheet" type="text/css" />
   <script src="/js/compiled/science.js" type="text/javascript"></script>
```

Comment:
@birdage all of that gets replaced with just this one. And for production we'll minify it.